web_editor/js/cards.js

```diff
@@ -11,7 +11,8 @@ export function makeLaneCard(lane, idx, app) {
   }
   node.appendChild(wrapInCenterDiv(width(lane)));
 
-  var finalRow = document.createElement("div");
+  var editRow = document.createElement("div");
+  editRow.align = "center";
```

Comment:
I'd prefer this done in CSS. `text-align: center` (for centering inline children of a block element) or `margin: 0 auto` (for centering a block element within its parent) will cover most cases.

Reply:
Done!